src/hera.cpp

```diff
@@ -50,6 +56,42 @@ struct hera_instance : evm_instance {
 
 namespace {
 
+#if HERA_EVM2WASM_JS
+// NOTE: assumes that pattern doesn't contain any formatting characters (e.g. %)
+string mktemp_string(string pattern) {
+  char tmp[PATH_MAX] = { 0 };
+  size_t len = snprintf(tmp, sizeof(tmp), "%s", pattern.c_str());
+  mktemp(tmp);
+  return string(tmp, len);
+}
+
+string evm2wasm(string const& input) {
```

Comment:
Need to add a comment:
a) expects and outputs a hex string
b) expects and outputs binary